ftw/footer/tests/test_footer_functional.py

```diff
@@ -0,0 +1,42 @@
+from ftw.footer.testing import FTW_FOOTER_FUNCTIONAL_TESTING
+from ftw.testbrowser import browsing
+from ftw.testbrowser.exceptions import HTTPServerError
+from plone.app.testing import setRoles
+from plone.app.testing import TEST_USER_ID
+from unittest import TestCase
+import transaction
+
+
+class TestFooterFunctional(TestCase):
+
+    layer = FTW_FOOTER_FUNCTIONAL_TESTING
+
+    def setUp(self):
+        super(TestFooterFunctional, self).setUp()
+        self.portal = self.layer['portal']
+        setRoles(self.portal, TEST_USER_ID, ['Manager'])
+        transaction.commit()
+
+    @browsing
+    def test_footer_on_non_contentish_object(self, browser):
+        # This tests checks tests if the footer is rendered on non contentish
+        # objects. For instance the view on an address portlet. But since we
+        # don't have such a portlet in default plone we use the ftw.footer
+        # column. The view itself throws an error, but we can still test
+        # the rendered footer.
+        browser.login().open(
+            self.portal,
+            {':action': "/++contextportlets++ftw.footer.column1/"
+                        "+/plone.portlet.static.Static"})
+        browser.fill({'Text The text to render': 'Blubber'}).submit()
+
+        try:
+            browser.open(
+                'http://nohost/plone/++contextportlets++ftw.footer.column1')
+        except HTTPServerError:
```

Comment:
As commented this is not that nice. But there is one view on every portlet. The `edit` view.

You may simply open `http://nohost/plone/++contextportlets++ftw.footer.column1/{ID_OF_PORTLET}/edit`